<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSvg</class>
 <widget class="QWidget" name="DlgSvg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVG Viewer &amp; Rasterizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/16x16_ico_png/svg_logo.png</normaloff>:/16x16_ico_png/svg_logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>SVG image viewer &amp;&amp; rasterizer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="imageFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Select SVG image file (none selected)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnSelectImage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resource/resource.qrc">
            <normaloff>:/16x16_ico_png/folder.png</normaloff>:/16x16_ico_png/folder.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Viewbox</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="cmbElement">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="viewX">
           <property name="minimum">
            <number>-100000</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="viewY">
           <property name="minimum">
            <number>-100000</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>W</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="viewW">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="viewH">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkViewAspect">
           <property name="text">
            <string>Keep aspect</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDoubleSize">
           <property name="text">
            <string>Size x2</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHalveSize">
           <property name="text">
            <string>Size /2</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="SvgViewWidget" name="view"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkShowBackground">
          <property name="text">
           <string>Show background</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkShowOutline">
          <property name="text">
           <string>Show outline</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Canvas</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="rasterWidth">
           <property name="maximum">
            <number>16384</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="rasterHeight">
           <property name="maximum">
            <number>16384</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDoubleCanvasSize">
           <property name="text">
            <string>Size x2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHalveCanvasSize">
           <property name="text">
            <string>Size /2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkRasterAspect">
           <property name="text">
            <string>Keep aspect</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkJson">
           <property name="text">
            <string>Generate JSON</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cmbFormat">
           <item>
            <property name="text">
             <string>PNG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>JPG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BMP</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Quality</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="quality">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveAs">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save as ...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SvgViewWidget</class>
   <extends>QGraphicsView</extends>
   <header>svg/svgview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>imageFile</tabstop>
  <tabstop>btnSelectImage</tabstop>
  <tabstop>viewX</tabstop>
  <tabstop>viewY</tabstop>
  <tabstop>viewW</tabstop>
  <tabstop>viewH</tabstop>
  <tabstop>chkViewAspect</tabstop>
  <tabstop>btnDoubleSize</tabstop>
  <tabstop>btnHalveSize</tabstop>
  <tabstop>view</tabstop>
  <tabstop>chkShowBackground</tabstop>
  <tabstop>chkShowOutline</tabstop>
  <tabstop>rasterWidth</tabstop>
  <tabstop>rasterHeight</tabstop>
  <tabstop>btnDoubleCanvasSize</tabstop>
  <tabstop>btnHalveCanvasSize</tabstop>
  <tabstop>chkRasterAspect</tabstop>
  <tabstop>cmbFormat</tabstop>
  <tabstop>quality</tabstop>
  <tabstop>btnSaveAs</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>cmbElement</tabstop>
  <tabstop>chkJson</tabstop>
 </tabstops>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
